modules/ROOT/pages/indexes/semantic-indexes/full-text-indexes.adoc

```diff
@@ -31,13 +31,20 @@ CREATE (nilsE:Employee {name: "Nils-Erik Karlsson", position: "Engineer", team:
 == Create full-text indexes
 
 Full-text indexes are created with the `CREATE FULLTEXT INDEX` command.
-An index can be given a unique name when created, which is used to reference the index when querying or dropping it.
+It is recommended to to give the index a name when it is created.
 If no name is given when created, a random name will be assigned to the full-text index.
-When creating a full-text index, you need to specify the labels/relationship types and property names it should apply to.
+
+The `CREATE FULLTEXT INDEX` command is optionally idempotent.
+This mean that its default behavior is to throw an error if an attempt is made to create the same index twice.
+If `IF NOT EXISTS` is appended to the command, no error is thrown and nothing happens should an index with the same name or same schema and index type already exist.
+It may still throw an error if conflicting constraints exist, such as constraints with the same name or schema and backing index type.
+As of Neo4j 5.16, the index name can also be given as a parameter, `CREATE FULLTEXT INDEX $name FOR ...`.
 
 [TIP]
 Creating a full-text index requires the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/database-administration/#access-control-database-administration-index[`CREATE INDEX` privilege].
 
+When creating a full-text index, you need to specify the labels/relationship types and property names it should apply to.
+
 This statement creates a full-text index named `namesAndTeams` on each `name` and `team` property for nodes with the label `Employee` or `Manager`:
 
 .Create a full-text index on a node label and property combination
```

Comment:
My original comment on this file was the link on line 72 being wrong, that doesn't seem to have been fixed